### ISSUE TITLE
Add. super45.fm

### ISSUE THREAD

**Issue by akrasiko** [web-scrobbler/web-scrobbler#3260]
Hi, 

I really enjoy Web Scrobbler and I would like to ask if the site super45.fm can be added to Web Srobbler. It's a cool online radio from Chile.

Thanks!

**Comment by hornbuckle** [comment]
Looks like this site just uses a Radio.co embed, which is already supported. Click the "Pop out player" icon in the top right corner of the player, and it will scrobble with the extension.

**Comment by akrasiko** [comment]
Hi David!

I hope this email finds you well. I tried what you said and Firefox
recognize the site and the song but it doesn't get scrobbled. When I click
on the Web Srobbler logo I got this message:

It looks like something went wrong while scrobbling the song. Open the
options page and try re-authenticating with the services. If
re-authentication doesn't help, and you still see this message, create a
new issue on the project page on GitHub.

Thanks for all your help and the amazing extension that Web Scrobbler is.
Warm regards!
